dockerfiles/Dockerfile.cross

```diff
@@ -0,0 +1,22 @@
+
+FROM    golang:1.8.1
+
+# allow replacing httpredir or deb mirror
+ARG     APT_MIRROR=deb.debian.org
+RUN     sed -ri "s/(httpredir|deb).debian.org/$APT_MIRROR/g" /etc/apt/sources.list
+
+RUN     apt-get update -qq && apt-get install -y -q \
+            libltdl-dev \
+            gcc-mingw-w64 \
+            parallel \
+            ;
```

Comment:
It's necessary because of the `\` on the previous line.  I like this style because it lets you add items to the list, and keep it sorted, without worrying about the special case of no `\` on the last line.

It's a minor thing though, and I'm fine to change it if you prefer I drop the `\` and `;`